src/lib/converter/converter.ts

```diff
@@ -310,49 +310,20 @@ export class Converter extends ChildableComponent<Application, ConverterComponen
         context.visitStack = oldVisitStack.slice();
         context.visitStack.push(node);
 
-        let result: Reflection;
+        let result: Reflection
```

Comment:
Why remove the `;`?

Reply:
Seems like an accident, thanks for pointing this out!